.env

```diff
@@ -0,0 +1 @@
+PORT=3000
```

Comment:
**THIS SHOULD BE ADDED TO YOUR .GITIGNORE**

Just put ``.env`` at the top of your .gitignore.